DeveloperConsole.cs

```diff
@@ -212,9 +247,65 @@ private void CommandRunner(string command)
                             case "level":
                                 result = $"Player {args[1]}'s Level: {LC_Lib.GetLevelNumber(Player)}";
                                 break;
+                            // Returns the player's look sensitivity
+                            case "look_sensitivity":
+                                result = $"Player {args[1]}'s Look Sensitivity: {LC_Lib.GetLookSensitivity(Player)}";
+                                break;
+                            default:
+                                result = "Error: Invalid arguments";
+                                break;
                         }
                     }
                     break;
+
+                // drop_all_items <player>
+                case "drop_all_held_items":
```

Comment:
Not implementing other items outside set and get commands at the moment. Please focus on those instead.